### DISCUSSION TITLE
How to force recognition of n speakers?

### DISCUSSION THREAD

**Discussion by rohantib** [pyannote/pyannote-audio#686] [Q&A]
I have mono, 8kHz GSM 6.10 call center audio with two speakers. I'm looking to separate the audios of the two speakers. For most of the audio files, pyannote fails to recognize that there are exactly two speakers; a lot of them are recognized to have only 1 speaker, and several have 3+ according to pyannote. Is there any way I can force the clustering to recognize 2 speakers? If not directly, are there any hyperparameters I could tune to achieve this?

**Comment by hbredin** [comment]
This issue might help: https://github.com/pyannote/pyannote-audio/issues/573
